external-crates/move/crates/move-compiler/src/diagnostics/mod.rs

```diff
@@ -747,6 +747,17 @@ macro_rules! diag {
             std::iter::empty::<String>(),
         )
     }};
+    ($code: expr, $primary: expr, note: $($note: expr),* $(,)?) => {{
```

Comment:
This isn't quite what I would have anticipated since it doesn't allow for secondary labels. Maybe this would be best suited for a different PR